starboard/build/toolchain/linux/BUILD.gn

```diff
@@ -37,6 +36,3 @@ clang_toolchain("x86") {
     is_clang_16 = true
   }
 }
-
-clang_lld_toolchain("clang_lld") {
```

Comment:
This is used for at least the modular build so we'll probably have to create another basic clang toolchain without explicit os/cpu references that BUILDCONFIG can point to for modular build

Reply:
But isn't it used because of the way Evergreen setup the flags?
Do we really need to use it?

Reply:
We don't want an lld toolchain, but we want a general clang toolchain that doesn't set any os/cpu/config path etc., just the clang_base_path

Reply:
Yeah agree. The flags are shared.